docs/pages/blog/2021-developer-survey-results.md

```diff
@@ -0,0 +1,406 @@
+
+This year we experimented using Typeform to run the survey, which allowed developers to rank the criteria available - something that wasn't possible with Google Forms. That way, we didn't have to limit the answers to 3 options, allowing to capture a richer perspective. But, even so, the results weren’t too different from last year. Design (look and feel) remains the most important criterion for choosing UI libraries, and customization and documentation quality are still there in the top 3. The notable difference this year seems to be performance climbing its way up, as it has been ranked as the fifth most important aspect.
+
+### What else can we do to improve MUI for you?
+
+<img src="/static/blog/2021-survey/6.png" style="width: 796px; margin-top: 16px; margin-bottom: 8px;" alt="Word cloud of what we can do to improve MUI" />
+
+<p class="blog-description">1007 out of 1589 answered this question in 2021</p>
+
+Here are the most recurring topics:
+
+- **More components:** We continue to see requests for more advanced components, following last year's pattern, such as Charts, Forms, and Calendar, to name a few.
+- **More examples:** Knowing the "officially recommended" ways of using any product is always helpful. We know that many of the learning resources you find out there right now are outdated (based on v4), so that's absolutely worth of investing.
+- **Provide more themes:** Even with the launch of Material v3, Material Design has long been a design language that some find outdated. That's why we are working on a second design system, thought to offer everything that the current Material package does (component quantity and quality) but with an alternative design direction.
+- **Fewer breaking changes:** MUI Core v5 definitely introduced some important breaking changes, especially because of the new styling solution. We’ll continue to explore the v5 migration topic across the post but, for now, it's good to share that we don’t expect to release any major this year. In fact, we aim for majors happening with at least 12 months apart from each other.
+- **Improve customization:** Many of the entries for customization ask for making it easier, to add more examples of common customization use-cases (like font-family and primary/secondary color change), and improvements to theme capabilities. Even with the popularity of emotion and styled-components, there continues to be an enourmous need to facilitate customization.
+
+If you have requests or ideas for improving the above-mentioned topics, here are a few tips to have them prioritized faster:
+
+- In both MUI Core and X repositories, you should label a request issue with `Waiting for upvotes`. If your issue gains a significant amount of upvotes, it will definitely get our attention and consideration.
```

Comment:
> you should label a request issue

Maybe we can rephrase this part, Since users cannot label issues?

Reply:
```suggestion
- In both MUI Core and X repositories, we'll label issues with feature/component requests with `Waiting for upvotes`. Make sure to structure, research and justify the request so it resonates with more developers, and, eventually, gets enough priority to be implemented.
```

I tend to forget the community doesn't have the same permissions as we do 😅 Here's a rephrased version considering both of your points. Let me know what you think!

Reply:
Looks great!

Reply:
Suggesting some revisions for grammar and clarity:

-In both MUI Core and X repositories, we'll label issues requesting features or components as `Waiting for upvotes`. So, make sure to structure, research and justify the request so it resonates with more developers, and, eventually, gets enough priority to be implemented.

+When you create an issue to request features or components in the MUI Core or X repositories, we'll label it as `Waiting for upvotes`. The more votes your issue receives, the higher a priority it will become—so make sure to structure, research, and justify your request so it resonates with other community members.